src/StackExchange.Redis/ServerEndPoint.cs

```diff
@@ -530,6 +530,13 @@ internal int LastInfoReplicationCheckSecondsAgo
             get { return unchecked(Environment.TickCount - Thread.VolatileRead(ref lastInfoReplicationCheckTicks)) / 1000; }
         }
 
+        internal bool MarkedForForceReplicationCheck
+        {
+            // TODO: consider multi threading
```

Comment:
Instead of a property here you can make this a `volatile bool` field :)

Reply:
Sure. I will make the change.